packages/wallet/src/wallet.ts

```diff
@@ -196,4 +210,37 @@ export default class Wallet extends AbstractWallet {
     const privateKey = Signer.generatePrivateKey(generateOptions?.entropy);
     return new Wallet(privateKey, generateOptions?.provider);
   }
+
+  /**
+   * Create wallet from a seed
+   */
+  static fromSeed(seed: string, path?: string): Wallet {
+    const hdWallet = HDWallet.fromSeed(seed);
+    const childWallet = hdWallet.derivePath(path || Wallet.defaultPath);
+    const wallet = new Wallet(<string>childWallet.privateKey);
```

Comment:
stylistically i think we can just return here

Reply:
I made the change makes sense